### PR TITLE
Fix packaging to remove `restart.sh` script

### PR DESCRIPTION
**Pull request by Shadowghost**
Fixes packaging issues occurring due to #9100

**Changes**
* Removes all references to `restart.sh` from packaging

Supercedes #9119

### CODE REVIEW

**Review by cvium — APPROVED** [review]

**Review by crobibero — APPROVED** [review]

**Review by Bond-009 — APPROVED** [review]

**Review by joshuaboniface — APPROVED** [review]
Goodbye `restart.sh`; you will not be missed.

File: fedora/jellyfin.service [hunk 1]
```diff
@@ -8,7 +8,7 @@ EnvironmentFile = /etc/sysconfig/jellyfin
 User = jellyfin
 Group = jellyfin
 WorkingDirectory = /var/lib/jellyfin
-ExecStart = /usr/bin/jellyfin $JELLYFIN_WEB_OPT $JELLYFIN_RESTART_OPT $JELLYFIN_FFMPEG_OPT $JELLYFIN_SERVICE_OPT $JELLYFIN_NOWEBAPP_OPT $JELLYFIN_ADDITIONAL_OPTS
```
Comment (sleepy-nols) [inline fedora/jellyfin.service:11]: seems like there was removed too much here. `$JELLYFIN_FFMPEG_OPT` is not supposed to be deleted.

Comment (crobibero) [reply fedora/jellyfin.service:11]: You replied to a 2 year old PR for a platform that the Jellyfin team does not support (Fedora).

Comment (sleepy-nols) [reply fedora/jellyfin.service:11]: oh sorry, I was browsing the mr branch instead of jellyfin master :/

**Pull request merged**